mods/screwdriver/init.lua

```diff
@@ -80,6 +130,7 @@ screwdriver.handler = function(itemstack, user, pointed_thing, mode, uses)
 	if should_rotate then
 		node.param2 = new_param2
 		minetest.swap_node(pos, node)
+		minetest.check_for_falling(pos)
```

Comment:
Well, other nodes can be attached as well, but they always stick to the bottom node, so the check can be omitted. However I do not see that as a performance penalty.